intranet/static/js/dashboard/eighth-widget.js

```diff
@@ -47,4 +46,4 @@ $(document).ready(function() {
     }
 
     sponsor_schedule_bind();
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2743148443/issues/source?bid=3176160&fileBranchId=3355598#l49)